doc/dev_guide/release_management.rst

```diff
@@ -10,79 +10,79 @@ Release management
 Release policy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A Tarantool release is identified by three digits, for example, 1.7.7.
-We use these digits according to their definitions provided at http://semver.org:
+A Tarantool release is identified by three digits, for example, 1.10.7:
 
-* The first digit stands for MAJOR release. A **major** release may contain
+* The first digit stands for ?a MAJOR release. A MAJOR release may contain
   *incompatible changes*.
-* The second digit stands for MINOR release, it does not contain incompatible
+* The second digit stands for ?a MINOR release. It does not contain incompatible
   changes, and is used for introducing backward-compatible *features*.
```

Comment:
Nope. We may introduce a backward-incompatible in a MINOR release.

Reply:
re-written